src/main/java/newbank/server/Customer.java

```diff
@@ -40,15 +40,31 @@ public String getUsername() {
   public String getPassword() {
     return password;
   }
-  
+
+  public boolean hasAccount(final String accountName) {
+    return accounts
+        .stream()
+        .filter(a -> a.getName().equalsIgnoreCase(accountName))
+        .findFirst()
+        .isPresent();
+  }
+
+  public Account getAccount(final String accountName) {
+    return accounts
+        .stream()
+        .filter(a -> a.getName().equalsIgnoreCase(accountName))
+        .findFirst()
+        .get();
```

Comment:
Note that `Optional#get` is being called without first checking whether `Optional#present`. We're able to do this without anything breaking because of the rework I talk about in `DepositCommand`.

Reply:
Done as part of the refactoring above.